src/Dotnet5.GraphQL3.Repositories.Abstractions/Pages/PagedResult.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+
+namespace Dotnet5.GraphQL3.Repositories.Abstractions.Pages
+{
+    public class PagedResult<T>
```

Comment:
Codacy found an issue: [This class can't be instantiated; make its constructor 'public'.](https://app.codacy.com/gh/AntonioFalcao/Dotnet5.GraphQL3.WebApplication/pullRequest?prid=6412839)